pkg/controller/nodelifecycle/scheduler/taint_manager.go

```diff
@@ -465,12 +469,12 @@ func (tc *NoExecuteTaintManager) handleNodeUpdate(ctx context.Context, nodeUpdat
 	}
 
 	// Create or Update
-	klog.V(4).InfoS("Noticed node update", "node", nodeUpdate)
+	logger.V(4).Info("Noticed node update", "node", nodeUpdate)
```

Comment:
Here the value for "node" is not the usual object. `nodeUpdateItem` is just a wrapper for a node name, you can log as an object without any loss of information.

Same above.

Reply:
done